ui/client/reducers/ui.js

```diff
@@ -1,21 +1,51 @@
-import _ from "lodash"
-import Dialogs from "../components/modals/Dialogs";
+// @flow
```

Comment:
A lot of unnecessary commas in this file

Reply:
"Trailing commas simplify adding and removing items to objects and arrays, since only the lines you are modifying must be touched. Another argument in favor of trailing commas is that it improves the clarity of diffs when an item is added or removed from an object or array:" https://eslint.org/docs/rules/comma-dangle